PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamCorrHists.h

```diff
@@ -206,6 +213,7 @@ class AliFemtoDreamCorrHists {
   TH2F **fPtQADist;
   TH2F **fMassQADistPart1;
   TH2F **fMassQADistPart2;
+  TH1F **fPairInvMassQAD;
```

Comment:
since you added a member variable, remember to increment the ClassDef counter at the end of the file
ClassDef(AliFemtoDreamCorrHists,7) 
-> ClassDef(AliFemtoDreamCorrHists,8)